src/Test/winswTests/YamlExtensionTest.cs

```diff
@@ -0,0 +1,70 @@
+﻿using NUnit.Framework;
+using WinSW;
+using WinSW.Configuration;
+using WinSW.Extensions;
+using WinSW.Plugins.RunawayProcessKiller;
+using WinSW.Util;
+using winswTests.Extensions;
+
+namespace winswTests
+{
+    public class YamlExtensionTest : ExtensionTestBase
+    {
+        private IWinSWConfiguration _testServiceDescriptor { get; set; }
+
+        readonly string testExtension = GetExtensionClassNameWithAssembly(typeof(RunawayProcessKillerExtension));
+
+        [SetUp]
+        public void SetUp()
+        {
+
+            string yamlDoc = $@"id: SERVICE_NAME
+name: Jenkins Slave
+description: This service runs Jenkins automation server.
+executable: java
+arguments: -Xrs  -jar \""%BASE%\slave.jar\"" -jnlpUrl ...
```

Comment:
```suggestion
arguments: -Xrs  -jar \""%BASE%\\agent.jar\"" -jnlpUrl ...
```

Reply:
Also remove the backslashes before quotes.